composer.json

```diff
@@ -67,7 +67,8 @@
         "twig/twig": "3.3.4",
         "vlucas/phpdotenv": "5.4.0",
         "waryway/php-traits-library": "1.0.4",
-        "yubico/u2flib-server": "1.0.2"
+        "yubico/u2flib-server": "1.0.2",
+        "openemr/oe-module-cqm": "dev-main"
```

Comment:
Not sure we want the tool in production version. Really should only be installed when needed by admins for reporting period updates on new releases.